recipe/meta.yaml

```
@@ -1,42 +1,81 @@
 {% set name = "httplib2" %}
-{% set version = "0.19.1" %}
-
+{% set version = "0.22.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/httplib2-{{ version }}.tar.gz
-  sha256: 0b12617eeca7433d4c396a100eaecfa4b08ee99aa881e6df6e257a7aad5d533d
+  url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f0463bc04d2546325eaba1da15f8e45763ed2a52b47c0331c721f1c85470c9ca
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
 
 requirements:
   host:
-    - python >=3
+    - python
     - pip
-    - pytest
+    - setuptools >=40.8.0
+    - wheel
   run:
-    - python >=3
+    - python
     - pyparsing >=2.4.2,<3
```

```suggestion
    - pyparsing >=2.4.2,<4
```
https://github.com/httplib2/httplib2/blob/v0.22.0/requirements.txt#L2